README.md

```diff
@@ -43,6 +43,18 @@ $ bundle install
 $ npm install
 ```
 
+## Run test on project with
+
+```bash
+$ rspec
+  - this will run all the test cases
+```
+
+```bash
+$ rspec spec/name_of_test_file.rb
```

Comment:
- [ ] [OPTIONAL] Well done implementing the recommended changes 👍🏽  However, this need to be corrected to the right path which is:
```
rspec spec/folder_name/name_of_test_file.rb
````

Reply:
Thank you @Whoistolu for being amazing!!!